stable/kommander/values.yaml

```diff
@@ -170,6 +170,16 @@ grafana:
     #   hosts:
     #   - grafana.example.com
 
+  folders: 
+    enabled: true
+    default: 
+      namespace: default-folder
```

Comment:
Since namespaces are cluster scope, I'm wondering if this should be more product-specific like kommander-default-folder ?